services/time/TUTORIAL.md

```diff
@@ -122,12 +122,12 @@ To obtain local, reliable time external solutions like [tlsdate][],
   local time of the validator nodes.
 
 The service implements only one transaction consisting of the actual
-validator’s time and signed with its key. The logic of such transaction
+validators time and signed with its key. The logic of such transaction
```

Comment:
"validators time" - "time of the validator" or leave as it was

Reply:
@Maria-Nosyk, I think we mean time of a single validator here. Btw, I don't think it's grammatically correct to use plural noun adjuncts